src/YouTubeSubtitlesExtractor/Abstractions/IYouTubeVideo.cs

```diff
@@ -28,4 +28,11 @@ public interface IYouTubeVideo
     /// <param name="options"><see cref="VideoOptions"/> instance.</param>
     /// <returns>Returns the list of <see cref="Subtitle"/> instances.</returns>
     Task<List<Subtitle>> ExtractSubtitlesAsync(VideoOptions options);
+
+    /// <summary>
+    /// Extracts the detail from the given video options.
```

Comment:
- Can you use "details" instead of "detail"?
- We don't use video options, but use video URL.